### PR TITLE
Testasd

### PR DESCRIPTION
**Pull request by cafesanu**
asd

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/cafesanu/ReviewNinja-Welcome/pull/2" target="_blank">ReviewNinja

File: Test-kim [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Yo this shit does not work
+Yo this shit does work
+Yo this shit maybe work
+Yo this shit might work
+Yo this shit might not work
```
Comment (cafesanu) [inline Test-kim:5]: I don't like this

File: Test-kim [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Yo this shit will not workYo this shit does not work
+Yo this shit does work
+Yo this shit maybe work
+Yo this shit might work
+Yo this shit might not work
```
Comment (cafesanu) [inline Test-kim:10]: I don't like it

File: Test-kim [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Yo this shit will not workYo this shit does not work
+Yo this shit does work
```
Comment (cafesanu) [inline Test-kim:17]: Fix this shit

File: Test-kim (hunk 1 above)
Comment (cafesanu) [inline Test-kim:5]: !fixed

File: Test-kim (hunk 2 above)
Comment (cafesanu) [inline Test-kim:10]: !resolved

File: Test-kim (hunk 3 above)
Comment (cafesanu) [inline Test-kim:17]: !fixed

File: Test-kim [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+Yo this shit maybe work
```
Comment (cafesanu) [inline Test-kim:23]: inoj

File: Test-kim [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+Yo this shit might work
+Yo this shit might not work
+Yo this shit will not workYo this shit does not work
```
Comment (cafesanu) [inline Test-kim:21]: !fix

Comment (cafesanu) [inline Test-kim:21]: !resolved

Comment (cafesanu) [inline Test-kim:21]: zdsafkhjodi j !fix

Comment (cafesanu) [inline Test-kim:21]: !fixed

File: Test-kim [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+Yo this shit does work
```
Comment (cafesanu) [inline Test-kim:12]: dalkhjsfoi asd !fix

Comment (cafesanu) [inline Test-kim:12]: I solved this issue !fixed

File: Test-kim (hunk 4 above)
Comment (cafesanu) [inline Test-kim:23]: !resolve

Comment (cafesanu) [inline Test-kim:23]: !fixed

Comment (cafesanu) [inline Test-kim:23]: !!fix

**Pull request merged**